spacecmd/src/lib/softwarechannel.py

```diff
@@ -630,6 +630,26 @@ def do_softwarechannel_listerrata(self, args):
 
 ####################
 
+def help_softwarechannel_listarches(self):
+    print("softwarechannel_listarches: lists the potential software"
```

Comment:
There is missing parenthesis on line 634.

Reply:
thanks.